readme.md

```diff
@@ -313,6 +313,7 @@
 	- [H2O](https://github.com/h2oai/awesome-h2o#readme) - Open source distributed machine learning platform written in Java with APIs in R, Python, and Scala.
 	- [Software Engineering for Machine Learning](https://github.com/SE-ML/awesome-seml#readme) - From experiment to production-level machine learning.
 	- [AI in Finance](https://github.com/georgezouq/awesome-ai-in-finance#readme) - Solving problems in finance with machine learning.
+	- [Production ML](https://github.com/EthicalML/awesome-production-machine-learning/#readme) - Frameworks to deploy, monitor & scale machine learning
```

Comment:
```suggestion
	- [Production ML](https://github.com/EthicalML/awesome-production-machine-learning/#readme) - Frameworks to deploy, monitor & scale machine learning.
```

You may need to add a period to the end of your description here. Also the description may need to be more like

```
Deploy, monitor & scale machine learning with frameworks.
```

Reply:
Thanks @keller-mark, I've added a period